ow_plexil/src/plans/FaultHandlingPattern4.plp

```diff
@@ -0,0 +1,142 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Pattern summary: Execute a sequence of actions. If a fault is injected
+// during a related action, the plan will stop the action. If/when the fault
```

Comment:
The phrase "the plan will stop the action" is not accurate, as it is the simulator that stops the action.  I suggest saying "the action will stop and end in the ABORTED state".